app/org/sagebionetworks/bridge/models/surveys/SurveyRule.java

```diff
@@ -133,12 +173,29 @@ public Builder withEndSurvey(Boolean endSurvey) {
             }
             return this;
         }
+        public Builder withDisplayIf(Boolean displayIf) {
+            if (Boolean.TRUE.equals(displayIf)) {
+                this.displayIf = displayIf;
+            }
```

Comment:
As written, once this is set to true, it can never be cleared. If we want false to be null, we need an else block to set it to null.